public/js/graph/react_node.js.jsx

```diff
@@ -174,93 +212,223 @@ var ReactRegion = React.createClass({
 });
 
 var ReactNodes = React.createClass({
-    getInitialState: function() {
+    getInitialState: function () {
         return {
             nodesList: [],
             hybridsList: []
         };
     },
     
-    componentDidMount: function() {
-        this.setState({nodesList:getNodes('.node')});
-        this.setState({hybridsList:getNodes('.hybrid')});  
+    componentDidMount: function () {
+        this.setState({nodesList: getNodes('.node')});
+        this.setState({hybridsList: getNodes('.hybrid')});  
     },
     
-    render: function() {
+    render: function () {
+        var svg = this.props.svg;
         return (
             <g id='nodes' stroke='black'>
-                {this.state.nodesList.map(function(entry, value) {
-                    return <ReactNode attributes={entry['attributes']} children={entry['children']} className='node' key={entry['id']} styles={entry['style']} hybrid={false}/>
-                })}
-                {this.state.hybridsList.map(function(entry, value) {
-                    return <ReactNode attributes={entry['attributes']} children={entry['children']} className='hybrid' key={entry['id']} styles={entry['style']} hybrid={true}/>
-                })}
+                {this.state.nodesList.map(function (entry, value) {
```

Comment:
This and the block below seem pretty similar, perhaps create a helper function to reduce redundancy?